.vib/grafana-tempo/vib-verify.json

```diff
@@ -22,11 +22,11 @@
           "url": "{SHA_ARCHIVE}",
           "path": "/bitnami/grafana-tempo"
         },
-        "runtime_parameters": "InF1ZXJ5RnJvbnRlbmQiOgogICJzZXJ2aWNlIjoKICAgICJwb3J0cyI6CiAgICAgICJodHRwIjogODAKICAgICJ0eXBlIjogIkxvYWRCYWxhbmNlciI=",
+        "runtime_parameters": "dGVtcG86CiAgZGF0YURpcjogL2JpdG5hbWkvZ3JhZmFuYS10ZW1wby9kYXRhCiAgdHJhY2VzOgogICAgamFlZ2VyOgogICAgICB0aHJpZnRIdHRwOiB0cnVlCiAgc2VhcmNoOgogICAgZW5hYmxlZDogdHJ1ZQogIGNvbnRhaW5lclBvcnRzOgogICAgd2ViOiAzMTAxCiAgICBncnBjOiA5MDk1CiAgICBnb3NzaXBSaW5nOiA3OTQ2CmNvbXBhY3RvcjoKICBlbmFibGVkOiB0cnVlCiAgcmVwbGljYUNvdW50OiAyCiAgcG9kU2VjdXJpdHlDb250ZXh0OgogICAgZW5hYmxlZDogdHJ1ZQogICAgZnNHcm91cDogMTAwMgogIGNvbnRhaW5lclNlY3VyaXR5Q29udGV4dDoKICAgIGVuYWJsZWQ6IHRydWUKICAgIHJ1bkFzVXNlcjogMTAwMgogIHNlcnZpY2U6CiAgICB0eXBlOiBMb2FkQmFsYW5jZXIKICAgIHBvcnRzOgogICAgICBodHRwOiA4MApkaXN0cmlidXRvcjoKICByZXBsaWNhQ291bnQ6IDEKaW5nZXN0ZXI6CiAgcmVwbGljYUNvdW50OiAxCnF1ZXJpZXI6CiAgcmVwbGljYUNvdW50OiAxCnF1ZXJ5RnJvbnRlbmQ6CiAgc2VydmljZToKICAgIHR5cGU6IExvYWRCYWxhbmNlcgogICAgcG9ydHM6CiAgICAgIGh0dHA6IDgwCnZ1bHR1cmU6CiAgZW5hYmxlZDogdHJ1ZQptZW1jYWNoZWQ6CiAgZW5hYmxlZDogdHJ1ZQpleHRyYURlcGxveToKLSBhcGlWZXJzaW9uOiBhcHBzL3YxCiAga2luZDogRGVwbG95bWVudAogIG1ldGFkYXRhOgogICAgbGFiZWxzOgogICAgICBhcHA6IHZpYi1zeW50aGV0aWMtbG9hZC1nZW5lcmF0b3IKICAgIG5hbWU6IHZpYi1zeW50aGV0aWMtbG9hZC1nZW5lcmF0b3IKICBzcGVjOgogICAgcmVwbGljYXM6IDEKICAgIHNlbGVjdG9yOgogICAgICBtYXRjaExhYmVsczoKICAgICAgICBhcHA6IHZpYi1zeW50aGV0aWMtbG9hZC1nZW5lcmF0b3IKICAgIHRlbXBsYXRlOgogICAgICBtZXRhZGF0YToKICAgICAgICBsYWJlbHM6CiAgICAgICAgICBhcHA6IHZpYi1zeW50aGV0aWMtbG9hZC1nZW5lcmF0b3IKICAgICAgc3BlYzoKICAgICAgICBjb250YWluZXJzOgogICAgICAgIC0gaW1hZ2U6IG9tbml0aW9uL3N5bnRoZXRpYy1sb2FkLWdlbmVyYXRvcjoxLjAuMjkKICAgICAgICAgIG5hbWU6IHN5bnRoZXRpYy1sb2FkLWdlbmVyYXRvcgogICAgICAgICAgZW52OgogICAgICAgICAgLSBuYW1lOiBKQUVHRVJfQ09MTEVDVE9SX1VSTAogICAgICAgICAgICB2YWx1ZTogaHR0cDovL2dyYWZhbmEtdGVtcG8tZGlzdHJpYnV0b3I6MTQyNjgK",
```

Comment:
I have some concerns about this chart's params:

- I'm not sure if we are checking that `tempo.containerPorts.web` is correctly modified. This could bechecked in `/bitnami/grafana-tempo/conf/tempo.yaml`. As this `tempo.yaml` file is a volumeMount, it should be checked regardless.
- Why are we not testing any of the `http/grpc` svc endpoints for `xxxx.service.ports.http` in Goss nor changing the default values?
- I think we are not testing whether Memcached is working.